### PR TITLE
Add run-tests.lisp and linux-test workflow.

### PR DESCRIPTION
**Pull request by Gleefre**

### CODE REVIEW

File: run-tests.lisp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env -S sbcl --script
+
+(require 'uiop)
+(require 'asdf)
+
+(let ((init-file (merge-pathnames ".sbclrc" (user-homedir-pathname))))
```
Comment (m-fedorov-s) [inline run-tests.lisp:6]: What is this for?
Maybe it is better to add `.sbclrc` to the repo?

Comment (Gleefre) [reply run-tests.lisp:6]: This is loading the sbcl's init file, which is not loaded because of `--script` toplevel option. It is needed to load quicklisp.

Comment (m-fedorov-s) [reply run-tests.lisp:6]: Isn't it possible to omit `--script` flag?

Comment (Gleefre) [reply run-tests.lisp:6]: It is, but then it wouldn't be possible to use `#!` in script file.

**Review by m-fedorov-s — APPROVED** [review]

**Pull request merged**